src/org/gk/gkCurator/GKCuratorFrame.java

```diff
@@ -384,9 +384,9 @@ else if (e.getPropertyName().equals("switchType")) {
 	                    GKInstance instance = (GKInstance) e.getNewValue();
 	                    GKSchemaClass oldCls = (GKSchemaClass) e.getOldValue();
 	                    schemaView.switchedType(oldCls, instance);
-	                    eventView.switchedType(oldCls, instance);
-	                    entityView.switchedType(oldCls, instance);
-	                    entityView.switchedType(oldCls, instance);
+	                    eventView.switchedType(instance);
```

Comment:
This change looks like more than just a javadoc change... what happened here?

Reply:
The method called here and in the two lines below was using `oldCls` as its first argument but on looking at the method body when editing its JavaDocs, I found the parameter was never used.  I removed it from the method so no JavaDocs for the param were needed and so had to change the callers.  I'm going to ask Guanming to review the PR and ask if this branch can be tested running the CuratorTool to make sure everything is okay before I merge.

Reply:
Ok, that makes sense. I haven't looked at the code in detail so I don't know if it's possible to test this easily with a unit test.